docs/application/dotnet/guides/user-interface/nui/text.md

```diff
@@ -312,10 +312,169 @@ The following markup elements are currently supported:
         Text="<font family='SamsungSans' weight='bold'>Hello world</font>"/>
     ```
 
+-   `<b>`
+
+    Sets Bold decoration for the characters inside the element.
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<b>Hello world</b>"/>
+    ```
+
+-   `<i>`
+
+    Sets Italic decoration for the characters inside the element.
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<i>Hello world</i>"/>
+    ```
+
+-   `<background>`
+
+    Sets the background color for the characters inside the element.
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<background color='yellow'>Hello world</background>"/>
+    ```
+
+-   `<u>`
+
+    Sets the underlined values for the characters inside the element.
+
+    The following attributes are supported:
+
+    -   `color`: The color of underline.
+    -   `height`: The height of underline.
+    -   `type`: The type of underline. The supported attribute values are `solid`, `dashed` and `double`.
+    -   `dash-gap`: The gap in pixels between the dashes of the dashed underline. Only valid when "DASHED" underline type is used.
+    -   `dash-width`: The width in pixels of the dashes of the dashed underline. Only valid when "DASHED" underline type is used.
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<u color='green' height='2.0f' type='dashed' dash-gap='2.0f' dash-width='3.0f'>Hello world</u>"/>
+    ```
+
+-   `<s>`
+
+    Sets the strikethrough values for the characters inside the element.
+
+    The following attributes are supported:
+
+    -   `color`: The color of strikethrough.
+    -   `height`: The height of strikethrough.
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<s color='green' height='2.0f' >Hello world</s>"/>
+    ```
+
+-   `<char-spacing>`
+
+    Sets the character spacing values for the characters inside the element.
+
+    Use the `value` attribute to define the spaces between characters in Pixels.
+
+    A positive value will make the characters far apart (expanded) and a negative value will bring them closer (condensed).
+
+    The following example shows how to apply it:
+
+    ```xaml
+    <base:TextLabel x:Name="txt" EnableMarkup="True"
+        Text="<char-spacing value='5.0f'>Hello world</char-spacing>"/>
+    ```
+
+-   `<p>`
+
+    Use paragraph tag to set many styles on paragraph's level for the lines inside the element.
+
+    The following attributes are supported:
+
+    -   `align`: Use the `align` attribute to define the horizontal alignment of paragraph. The supported attribute values are `begin`, `center` and `end` .
```

Comment:
Missing article before the word "paragraph"
```suggestion
    -   `align`: Use the `align` attribute to define the horizontal alignment of a paragraph. The supported attribute values are `begin`, `center` and `end` .
```

Reply:
Done.